src/LossFunctions.jl

```diff
@@ -76,23 +76,20 @@ export
     ZeroOneLoss,
 
     OrdinalMarginLoss,
-    OrdinalHingeLoss,
```

Comment:
Let's put this through a deprecation cycle

Reply:
Added depwarn :+1: